electrical/readme.MD

```diff
@@ -0,0 +1,261 @@
+Tamiya Connectors| E35 | 1 | <p align="center"> <img src="../images/components/electronics/E35.png" width="25%"> </p>
+
+![Battery and board connections](../images/electronics/battery_wiring.png)
+
+**Figure 4. Connecting the Battery to switch and board**
+ 
+# 3. Connecting Data Cables
+**NOTE**: Before proceeding, you need to have completed the assembly of the custom PCB board and populated all the components. If you have not yet done so, build the Control Board now by following the document at /Electrical/PCB Assembly.pdf.
+
+Next, we will next plug in all the cables that allow the various devices to communicate with each other.
+
+**Table 4. Necessary Parts**
+
+Item | Ref | Qty | Image 
+---- | --- | --- | ----- 
+Rectangular 6P 1x6 Jumper Cable | E26 | 1 | <p align="center"> <img src="../images/components/electronics/E26.png" width="25%"> </p>
+USB Power Cable | E28 | 1| <p align="center"> <img src="../images/components/electronics/E28.png" width="25%"> </p>
+Rectangular 40P 2x20 Ribbon Cable | E29 | 1 | <p align="center"> <img src="../images/components/electronics/E29.png" width="25%"> </p>
+Rectangular 16P Ribbon Cable | E30 | 1| <p align="center"> <img src="../images/components/electronics/E30.png" width="25%"> </p>
+ USB to TTL Serial Adapter | E48 | 1| <p align="center"> <img src="../images/components/electronics/E48.png" width="25%"> </p>
+ 
+
+ 
+1. Connect the E26 cable from the J10 Connector on the Control Board to the J5 Connector on the Arduino Shield
+1. Connect the E30 16 Position ribbon cable from the J1 Connector to the INPUT Connector on the LED Matrix
+1. Connect the E48 TTL serial to USB cable from any USB port on the Raspberry Pi to the 6 Position headers at J8. The pinout should match the table below (see Figure 5):
+
+**Table 5. J8 Pins and Wire Colors**
+
+Control Board PCB J8 pin | TTL Serial cable wire color
+------------------------ | ---------------------------
+GND | Black
+N/A | Not Connected
+5V | Red
+TxD | Green
+RxD | White
+N/A | Not Connected
+
+![USB-serial](../images/electronics/usb_ttl_serial.png)
+
+  **Figure 5. USB / TTL Serial Cable Connection**
+  
+# 4. Connecting Motors and Encoders
+Next, we will connect the motors and encoders to the RoboClaw motor controllers. For each of the 10 motors you will need to connect motor power as well as the encoders. This means that there will be close to 60 wires traveling between the motor controllers and the motors/encoders, so it is very important to try to keep your wires organized and bundled together well. Below is some information about strain relief and using wire braid in your electronics systems. See these links and Figures 6 and 7 for examples.
+
+* [Techopedia Strain Relief](https://www.techopedia.com/definition/2301/strain-relief) 
+* [YouTube Wire Braid](https://www.youtube.com/watch?v=FeCs98TSsYQ)
+
+![](Pictures/Routing/route1.PNG)![](../images/electronics/route2.png)![](Pictures/Routing/route3.PNG)
```

Comment:
scale this image down a bit, use html like `<img src="/images/body/9x12_Plate_cut2.png" height="350">`

see https://github.com/nasa-jpl/open-source-rover/blob/master/examples/README_example.md

Reply:
I left these big because they are helpful when running cabling.